backend/src/zango/apps/shared/tenancy/management/commands/ws_migrate.py

```diff
@@ -21,7 +21,15 @@ def add_arguments(self, parser):
         parser.add_argument("--package", help="Run the migrations for the package")
 
     def handle(self, *args, **options):
-        tenant_obj = TenantModel.objects.get(name=options["workspace"])
+        while True:
+            try:
+                tenant_obj = TenantModel.objects.get(name=options["workspace"])
+                break  # Exit the loop if a valid workspace is found
+            except TenantModel.DoesNotExist:
+                self.stdout.write(self.style.ERROR(f"The app name '{workspace}' provided as an argument is invalid. Please ensure that you have entered the correct app name and try again."))
```

Comment:
here also it will throw an error `UnboundLocalError: local variable 'workspace' referenced before assignment` for the first execution